packages/react/src/date-picker/Calendar.tsx

```diff
@@ -154,33 +155,39 @@ const calendarComponents: CustomComponents = {
 			classNames.nav_button_next,
 		].join(' ');
 
+		const isFirst = props.displayIndex === 0;
+		const isLast = props.displayIndex === displayMonths.length - 1;
```

Comment:
Not fan of these variable names, but it's what RDP uses in their versions, so want to make the translation easier